src/main/java/hudson/plugins/scm/koji/model/Build.java

```diff
@@ -41,6 +42,8 @@
     @XmlElementWrapper(name = "tags")
     @XmlElement(name = "tag")
     private final Set<String> tags;
+    private String downloadUrl;
```

Comment:
I'm not happy about these 2 properties. Until now it was nice, immutable class. These 2 properties breaks it.

Reply:
I didn't find better solution.